api/all/src/main/java/io/opentelemetry/api/trace/Span.java

```diff
@@ -22,6 +22,12 @@
  *
  * <p>Spans are created by the {@link SpanBuilder#startSpan} method.
  *
+ * <p>When inserting attributes, {@code null} keys and values are prohibited, and in that case the
```

Comment:
```suggestion
 * <p>When inserting attributes, {@code null} keys and values are prohibited, and if provided, the
```

Think this reads a bit better